plugins/spu2-x/src/Dma.cpp

```diff
@@ -291,14 +291,14 @@ void V_Core::PlainDMAWrite(u16 *pMem, u32 size)
         // Buffer doesn't wrap/overflow!
         // Just set the TDA and check for an IRQ...
 
-        TDA = (buff1end + 1) & 0xfffff;
+        TDA = buff1end;
```

Comment:
Is this change necessary too? It makes the TDA handling inconsistent (see code above).